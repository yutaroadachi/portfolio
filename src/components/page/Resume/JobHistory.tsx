import {
  Accordion,
  AccordionButton,
  AccordionIcon,
  AccordionItem,
  AccordionPanel,
  Box,
  chakra,
  Stack,
  Table,
  Tbody,
  Td,
  Th,
  Thead,
  Tr,
} from '@chakra-ui/react'
import React from 'react'

export const JobHistory = () => {
  return (
    <Box w="full">
      <chakra.h2 id="job-history" textStyle="h2WithTreat" color="main" mb={8}>
        職務経歴
      </chakra.h2>
      <ForPC />
      <ForSP />
    </Box>
  )
}

const ForPC = () => {
  return (
    <Stack display={{ base: 'none', md: 'flex' }} spacing={8}>
      {jobs.map((job, idx) => {
        return (
          <Stack spacing={4} key={idx}>
            <Box>
              <chakra.h3 textStyle="h3">{job.companyName}</chakra.h3>
            </Box>
            <Table colorScheme="blackAlpha" size="lg">
              <Thead>
                <Tr>
                  <Th whiteSpace="nowrap">期間</Th>
                  <Th>業務内容</Th>
                  <Th whiteSpace="nowrap">使用技術</Th>
                  <Th whiteSpace="nowrap">役割</Th>
                </Tr>
              </Thead>
              <Tbody>
                <Tr>
                  <Td whiteSpace="nowrap">
                    {job.start}〜
                    <br />
                    {job.end ?? '現在'}
                  </Td>
                  <Td>{job.detail}</Td>
                  <Td whiteSpace="nowrap">
                    {job.skills.map((skill) => (
                      <chakra.div key={skill}>{skill}</chakra.div>
                    ))}
                  </Td>
                  <Td whiteSpace="nowrap">
                    {job.roles.map((role) => (
                      <chakra.div key={role}>{role}</chakra.div>
                    ))}
                  </Td>
                </Tr>
              </Tbody>
            </Table>
          </Stack>
        )
      })}
    </Stack>
  )
}

const ForSP = () => {
  return (
    <Accordion
      display={{ base: 'block', md: 'none' }}
      defaultIndex={[0]}
      allowMultiple
    >
      <Stack spacing={4}>
        {jobs.map((job, idx) => {
          return (
            <AccordionItem key={idx}>
              <AccordionButton>
                <Box flex="1" textAlign="left">
                  <chakra.h3 textStyle="h3">{job.companyName}</chakra.h3>
                  <chakra.div fontSize="14px" color="gray.500">
                    {job.start}〜{job.end ?? '現在'}
                  </chakra.div>
                </Box>
                <AccordionIcon />
              </AccordionButton>
              <AccordionPanel>
                <Stack spacing={4}>
                  <chakra.p>{job.detail}</chakra.p>
                  <Box>
                    <chakra.div fontWeight="bold">使用技術</chakra.div>
                    {job.skills.map((skill) => {
                      return <chakra.div key={skill}>{skill}</chakra.div>
                    })}
                  </Box>
                  <Box>
                    <chakra.div fontWeight="bold">役割</chakra.div>
                    {job.roles.map((role) => {
                      return <chakra.div key={role}>{role}</chakra.div>
                    })}
                  </Box>
                </Stack>
              </AccordionPanel>
            </AccordionItem>
          )
        })}
      </Stack>
    </Accordion>
  )
}

const jobs: {
  companyName: string
  start: string
  end?: string
  detail: React.ReactNode
  skills: string[]
  roles: string[]
}[] = [
  {
    companyName: '金融系スタートアップ',
    start: '2020/9',
    detail:
      'くらしとお金の社会課題を解決するシステムとサービスを提供する会社にて、社内のファイナンシャルプランナー向けのCRM・SFAシステムやはたらく世代のお金の診断・相談サービスの開発に従事し、フロントエンド（Next.js, TypeScript）からバックエンド（Ruby on Rails）まで幅広く経験しています。また、チームリーダーとしてスクラム開発をリードする経験もしています。',
    skills: ['Next.js', 'TypeScript', 'Ruby on Rails'],
    roles: ['メンバー', 'チームリーダー'],
  },
  {
    companyName: '金融系スタートアップ',
    start: '2019/8',
    end: '2020/8',
    detail:
      '上場企業のIR部門と機関投資家、証券会社をつなぐプラットフォームを提供する会社にて、メインプロダクトの開発に従事し、フロントエンド（React）からバックエンド（Ruby on Rails）まで幅広く経験しました。また、営業同行やカスタマーサポート、企画などのビジネスサイドに近い経験もしました。',
    skills: ['React', 'Ruby on Rails'],
    roles: ['メンバー'],
  },
  {
    companyName: '大手SIer',
    start: '2018/4',
    end: '2019/7',
    detail:
      '大手SIerの会社にて、大手金融機関のシステム子会社に常駐して社内稟議などのペーパーレス化システムの改修・保守に従事し、リリース作業を自動化するBashスクリプトの開発や保守チームのリーダーを経験しました。',
    skills: ['Bash'],
    roles: ['メンバー', 'チームリーダー'],
  },
]
