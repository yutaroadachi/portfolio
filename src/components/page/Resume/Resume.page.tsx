import React, { ComponentProps } from 'react'
import { Layout } from 'src/components/ui/Layout'
import { MetaTag } from 'src/components/ui/MetaTag'
import { Resume } from './Resume'

export type ResumePageProps = ComponentProps<typeof Resume>

export const ResumePage = ({}: ResumePageProps) => {
  return (
    <Layout>
      <MetaTag
        title="履歴書"
        description="Yutaro Adachi(フロントエンドエンジニア)の履歴書です。"
      />
      <Resume />
    </Layout>
  )
}
