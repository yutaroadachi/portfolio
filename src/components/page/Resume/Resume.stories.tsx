import { Meta, Story } from '@storybook/react'
import React from 'react'
import { Resume, ResumeProps } from './Resume'

export default {
  title: 'page/Resume',
  component: Resume,
} as Meta<ResumeProps>

type Template = Story<ResumeProps>

const Template: Template = (args) => <Resume {...args} />

export const Default: Template = Template.bind({})
