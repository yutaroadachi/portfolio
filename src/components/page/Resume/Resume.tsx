import { ExternalLinkIcon } from '@chakra-ui/icons'
import { Box, chakra, Link, VStack } from '@chakra-ui/react'
import React from 'react'
import { Biography } from 'src/components/domain/Biography'
import { URL } from 'src/constants/url'
import { JobHistory } from './JobHistory'

export type ResumeProps = {}

export const Resume = ({}: ResumeProps) => {
  return (
    <Box>
      <chakra.h1 id="resume" textStyle="h1" color="main" mb={8}>
        履歴書
      </chakra.h1>
      <VStack spacing={8}>
        <JobHistory />
        <AcademicHistory />
        <Biography />
      </VStack>
    </Box>
  )
}

const AcademicHistory = () => {
  return (
    <Box w="full">
      <chakra.h2
        id="academic-history"
        textStyle="h2WithTreat"
        color="main"
        mb={8}
      >
        最終学歴
      </chakra.h2>
      <Box textAlign="center">
        <chakra.h3 textStyle="h3">岡山大学 経済学部</chakra.h3>
        <chakra.p fontSize="14px" color="gray.500">
          2014/4 ~ 2018/3
        </chakra.p>
        <Link href={URL.okayamaU} isExternal fontSize="12px" color="link">
          大学のHPを見る
          <ExternalLinkIcon pb="2px" ml="2px" />
        </Link>
      </Box>
    </Box>
  )
}
