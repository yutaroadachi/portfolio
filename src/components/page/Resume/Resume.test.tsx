import React from 'react'
import { render } from 'src/__tests__/utils'
import { Resume } from './Resume'

test('Resume', () => {
  const { container } = render(<Resume />)
  expect(container).toMatchSnapshot()
})
