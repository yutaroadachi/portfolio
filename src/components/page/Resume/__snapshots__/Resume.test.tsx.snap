// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume 1`] = `
<div>
  <div
    class="css-0"
  >
    <h1
      class="css-15mzpqr"
      id="resume"
    >
      履歴書
    </h1>
    <div
      class="chakra-stack css-aajx3r"
    >
      <div
        class="css-1ki54i"
      >
        <h2
          class="css-jjss8v"
          id="job-history"
        >
          職歴
        </h2>
        <div
          class="chakra-stack css-175wvn7"
        >
          <div
            class="chakra-stack css-egoftb"
          >
            <div
              class="css-0"
            >
              <h3
                class="css-1vpdv9u"
              >
                株式会社モニクル
              </h3>
            </div>
            <table
              class="chakra-table css-5605sr"
              role="table"
            >
              <thead
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <th
                    class="css-iz806w"
                  >
                    期間
                  </th>
                  <th
                    class="css-1tk997h"
                  >
                    業務内容
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    使用技術
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    役割
                  </th>
                </tr>
              </thead>
              <tbody
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    2020/9
                    〜
                    <br />
                    現在
                  </td>
                  <td
                    class="css-nzu2gk"
                    role="gridcell"
                  >
                    「はたらく世代」の資産運用をサポートする会社のファイナンシャルアドバイザー向けCRMシステムおよび、お客様向けサービスサイトの開発に従事し、フロントエンド開発からバックエンド開発まで幅広く経験しております。
                    <br />
                    <span
                      class="css-1ov1hvb"
                    >
                      ※株式会社モニクルは持株会社であり、正確には同グループ会社の
                      <a
                        class="chakra-link css-wmzrah"
                        href="https://onemile.co.jp/"
                        rel="noopener noreferrer"
                        target="_blank"
                      >
                        株式会社OneMile Partners
                        <svg
                          class="chakra-icon css-lvnjke"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <g
                            fill="none"
                            stroke="currentColor"
                            stroke-linecap="round"
                            stroke-width="2"
                          >
                            <path
                              d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                            />
                            <path
                              d="M15 3h6v6"
                            />
                            <path
                              d="M10 14L21 3"
                            />
                          </g>
                        </svg>
                      </a>
                      に入社後、同グループ会社の
                      <a
                        class="chakra-link css-wmzrah"
                        href="https://navipla.com/"
                        rel="noopener noreferrer"
                        target="_blank"
                      >
                        株式会社ナビゲータープラットフォーム
                        <svg
                          class="chakra-icon css-lvnjke"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <g
                            fill="none"
                            stroke="currentColor"
                            stroke-linecap="round"
                            stroke-width="2"
                          >
                            <path
                              d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                            />
                            <path
                              d="M15 3h6v6"
                            />
                            <path
                              d="M10 14L21 3"
                            />
                          </g>
                        </svg>
                      </a>
                      を経由して転籍した形になっておりますが、業務内容に変化はありません。
                    </span>
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      React
                    </p>
                    <p
                      class="css-0"
                    >
                      TypeScript
                    </p>
                    <p
                      class="css-0"
                    >
                      Next.js
                    </p>
                    <p
                      class="css-0"
                    >
                      Ruby on Rails
                    </p>
                    <p
                      class="css-0"
                    >
                      Go
                    </p>
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      メンバ
                    </p>
                  </td>
                </tr>
              </tbody>
            </table>
            <div
              class="css-0"
            >
              <a
                class="chakra-link css-1f26k5b"
                href="https://monicle.co.jp/"
                rel="noopener noreferrer"
                target="_blank"
              >
                会社のHPを見る
                <svg
                  class="chakra-icon css-soaf0h"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                    />
                    <path
                      d="M15 3h6v6"
                    />
                    <path
                      d="M10 14L21 3"
                    />
                  </g>
                </svg>
              </a>
            </div>
          </div>
          <div
            class="chakra-stack css-egoftb"
          >
            <div
              class="css-0"
            >
              <h3
                class="css-1vpdv9u"
              >
                株式会社みんせつ
              </h3>
            </div>
            <table
              class="chakra-table css-5605sr"
              role="table"
            >
              <thead
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <th
                    class="css-iz806w"
                  >
                    期間
                  </th>
                  <th
                    class="css-1tk997h"
                  >
                    業務内容
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    使用技術
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    役割
                  </th>
                </tr>
              </thead>
              <tbody
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    2019/8
                    〜
                    <br />
                    2020/8
                  </td>
                  <td
                    class="css-nzu2gk"
                    role="gridcell"
                  >
                    上場企業のIR部門と機関投資家、証券会社をつなぐプラットフォームの企画・開発に従事し、フロントエンド開発からバックエンド開発まで幅広く経験しました。
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      React
                    </p>
                    <p
                      class="css-0"
                    >
                      Ruby on Rails
                    </p>
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      メンバ
                    </p>
                  </td>
                </tr>
              </tbody>
            </table>
            <div
              class="css-0"
            >
              <a
                class="chakra-link css-1f26k5b"
                href="https://msetsu.com/"
                rel="noopener noreferrer"
                target="_blank"
              >
                会社のHPを見る
                <svg
                  class="chakra-icon css-soaf0h"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                    />
                    <path
                      d="M15 3h6v6"
                    />
                    <path
                      d="M10 14L21 3"
                    />
                  </g>
                </svg>
              </a>
            </div>
          </div>
          <div
            class="chakra-stack css-egoftb"
          >
            <div
              class="css-0"
            >
              <h3
                class="css-1vpdv9u"
              >
                NTTコムウェア株式会社
              </h3>
            </div>
            <table
              class="chakra-table css-5605sr"
              role="table"
            >
              <thead
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <th
                    class="css-iz806w"
                  >
                    期間
                  </th>
                  <th
                    class="css-1tk997h"
                  >
                    業務内容
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    使用技術
                  </th>
                  <th
                    class="css-iz806w"
                  >
                    役割
                  </th>
                </tr>
              </thead>
              <tbody
                class="css-0"
              >
                <tr
                  class="css-0"
                  role="row"
                >
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    2018/4
                    〜
                    <br />
                    2019/7
                  </td>
                  <td
                    class="css-nzu2gk"
                    role="gridcell"
                  >
                    某大手金融機関に常駐して社内稟議などのペーパーレス化システムの開発・保守に従事し、メンバとしてはテスト設計やリリース作業を自動化するbashスクリプトの製造を、保守チーム（5名程度）のリーダーとしては顧客折衝やマネジメント、資料作成を経験しました。
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      bash
                    </p>
                    <p
                      class="css-0"
                    >
                      SQL
                    </p>
                  </td>
                  <td
                    class="css-1s6o4al"
                    role="gridcell"
                  >
                    <p
                      class="css-0"
                    >
                      メンバ
                    </p>
                    <p
                      class="css-0"
                    >
                      チームリーダー
                    </p>
                  </td>
                </tr>
              </tbody>
            </table>
            <div
              class="css-0"
            >
              <a
                class="chakra-link css-1f26k5b"
                href="https://www.nttcom.co.jp/"
                rel="noopener noreferrer"
                target="_blank"
              >
                会社のHPを見る
                <svg
                  class="chakra-icon css-soaf0h"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <g
                    fill="none"
                    stroke="currentColor"
                    stroke-linecap="round"
                    stroke-width="2"
                  >
                    <path
                      d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                    />
                    <path
                      d="M15 3h6v6"
                    />
                    <path
                      d="M10 14L21 3"
                    />
                  </g>
                </svg>
              </a>
            </div>
          </div>
        </div>
        <div
          class="chakra-accordion css-259i0c"
        >
          <div
            class="chakra-stack css-egoftb"
          >
            <div
              class="chakra-accordion__item css-ejn068"
            >
              <button
                aria-controls="accordion-panel-1"
                aria-expanded="true"
                class="chakra-accordion__button css-df84mh"
                data-index="0"
                id="accordion-button-1"
                type="button"
              >
                <div
                  class="css-1eziwv"
                >
                  <h3
                    class="css-1vpdv9u"
                  >
                    株式会社モニクル
                  </h3>
                  <p
                    class="css-1vf9ext"
                  >
                    2020/9
                    〜
                    現在
                  </p>
                </div>
                <svg
                  aria-hidden="true"
                  class="chakra-icon chakra-accordion__icon css-j2ph2z"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    fill="currentColor"
                  />
                </svg>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: block; opacity: 0; height: auto;"
              >
                <div
                  aria-labelledby="accordion-button-1"
                  class="chakra-accordion__panel css-1hb5ode"
                  id="accordion-panel-1"
                  role="region"
                >
                  <div
                    class="chakra-stack css-egoftb"
                  >
                    <p
                      class="css-0"
                    >
                      「はたらく世代」の資産運用をサポートする会社のファイナンシャルアドバイザー向けCRMシステムおよび、お客様向けサービスサイトの開発に従事し、フロントエンド開発からバックエンド開発まで幅広く経験しております。
                      <br />
                      <span
                        class="css-1ov1hvb"
                      >
                        ※株式会社モニクルは持株会社であり、正確には同グループ会社の
                        <a
                          class="chakra-link css-wmzrah"
                          href="https://onemile.co.jp/"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          株式会社OneMile Partners
                          <svg
                            class="chakra-icon css-lvnjke"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <g
                              fill="none"
                              stroke="currentColor"
                              stroke-linecap="round"
                              stroke-width="2"
                            >
                              <path
                                d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                              />
                              <path
                                d="M15 3h6v6"
                              />
                              <path
                                d="M10 14L21 3"
                              />
                            </g>
                          </svg>
                        </a>
                        に入社後、同グループ会社の
                        <a
                          class="chakra-link css-wmzrah"
                          href="https://navipla.com/"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          株式会社ナビゲータープラットフォーム
                          <svg
                            class="chakra-icon css-lvnjke"
                            focusable="false"
                            viewBox="0 0 24 24"
                          >
                            <g
                              fill="none"
                              stroke="currentColor"
                              stroke-linecap="round"
                              stroke-width="2"
                            >
                              <path
                                d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                              />
                              <path
                                d="M15 3h6v6"
                              />
                              <path
                                d="M10 14L21 3"
                              />
                            </g>
                          </svg>
                        </a>
                        を経由して転籍した形になっておりますが、業務内容に変化はありません。
                      </span>
                    </p>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        使用技術
                      </p>
                      <p
                        class="css-0"
                      >
                        React
                      </p>
                      <p
                        class="css-0"
                      >
                        TypeScript
                      </p>
                      <p
                        class="css-0"
                      >
                        Next.js
                      </p>
                      <p
                        class="css-0"
                      >
                        Ruby on Rails
                      </p>
                      <p
                        class="css-0"
                      >
                        Go
                      </p>
                    </div>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        役割
                      </p>
                      <p
                        class="css-0"
                      >
                        メンバ
                      </p>
                    </div>
                    <a
                      class="chakra-link css-wmzrah"
                      href="https://monicle.co.jp/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      会社のHPを見る
                      <svg
                        class="chakra-icon css-soaf0h"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <g
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-width="2"
                        >
                          <path
                            d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                          />
                          <path
                            d="M15 3h6v6"
                          />
                          <path
                            d="M10 14L21 3"
                          />
                        </g>
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="chakra-accordion__item css-ejn068"
            >
              <button
                aria-controls="accordion-panel-2"
                aria-expanded="false"
                class="chakra-accordion__button css-df84mh"
                data-index="1"
                id="accordion-button-2"
                type="button"
              >
                <div
                  class="css-1eziwv"
                >
                  <h3
                    class="css-1vpdv9u"
                  >
                    株式会社みんせつ
                  </h3>
                  <p
                    class="css-1vf9ext"
                  >
                    2019/8
                    〜
                    2020/8
                  </p>
                </div>
                <svg
                  aria-hidden="true"
                  class="chakra-icon chakra-accordion__icon css-186l2rg"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    fill="currentColor"
                  />
                </svg>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-2"
                  class="chakra-accordion__panel css-1hb5ode"
                  id="accordion-panel-2"
                  role="region"
                >
                  <div
                    class="chakra-stack css-egoftb"
                  >
                    <p
                      class="css-0"
                    >
                      上場企業のIR部門と機関投資家、証券会社をつなぐプラットフォームの企画・開発に従事し、フロントエンド開発からバックエンド開発まで幅広く経験しました。
                    </p>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        使用技術
                      </p>
                      <p
                        class="css-0"
                      >
                        React
                      </p>
                      <p
                        class="css-0"
                      >
                        Ruby on Rails
                      </p>
                    </div>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        役割
                      </p>
                      <p
                        class="css-0"
                      >
                        メンバ
                      </p>
                    </div>
                    <a
                      class="chakra-link css-wmzrah"
                      href="https://msetsu.com/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      会社のHPを見る
                      <svg
                        class="chakra-icon css-soaf0h"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <g
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-width="2"
                        >
                          <path
                            d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                          />
                          <path
                            d="M15 3h6v6"
                          />
                          <path
                            d="M10 14L21 3"
                          />
                        </g>
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="chakra-accordion__item css-ejn068"
            >
              <button
                aria-controls="accordion-panel-3"
                aria-expanded="false"
                class="chakra-accordion__button css-df84mh"
                data-index="2"
                id="accordion-button-3"
                type="button"
              >
                <div
                  class="css-1eziwv"
                >
                  <h3
                    class="css-1vpdv9u"
                  >
                    NTTコムウェア株式会社
                  </h3>
                  <p
                    class="css-1vf9ext"
                  >
                    2018/4
                    〜
                    2019/7
                  </p>
                </div>
                <svg
                  aria-hidden="true"
                  class="chakra-icon chakra-accordion__icon css-186l2rg"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                    fill="currentColor"
                  />
                </svg>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-3"
                  class="chakra-accordion__panel css-1hb5ode"
                  id="accordion-panel-3"
                  role="region"
                >
                  <div
                    class="chakra-stack css-egoftb"
                  >
                    <p
                      class="css-0"
                    >
                      某大手金融機関に常駐して社内稟議などのペーパーレス化システムの開発・保守に従事し、メンバとしてはテスト設計やリリース作業を自動化するbashスクリプトの製造を、保守チーム（5名程度）のリーダーとしては顧客折衝やマネジメント、資料作成を経験しました。
                    </p>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        使用技術
                      </p>
                      <p
                        class="css-0"
                      >
                        bash
                      </p>
                      <p
                        class="css-0"
                      >
                        SQL
                      </p>
                    </div>
                    <div
                      class="css-0"
                    >
                      <p
                        class="css-1vg6q84"
                      >
                        役割
                      </p>
                      <p
                        class="css-0"
                      >
                        メンバ
                      </p>
                      <p
                        class="css-0"
                      >
                        チームリーダー
                      </p>
                    </div>
                    <a
                      class="chakra-link css-wmzrah"
                      href="https://www.nttcom.co.jp/"
                      rel="noopener noreferrer"
                      target="_blank"
                    >
                      会社のHPを見る
                      <svg
                        class="chakra-icon css-soaf0h"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <g
                          fill="none"
                          stroke="currentColor"
                          stroke-linecap="round"
                          stroke-width="2"
                        >
                          <path
                            d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                          />
                          <path
                            d="M15 3h6v6"
                          />
                          <path
                            d="M10 14L21 3"
                          />
                        </g>
                      </svg>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="css-1ki54i"
      >
        <h2
          class="css-jjss8v"
          id="academic-history"
        >
          最終学歴
        </h2>
        <div
          class="css-xi606m"
        >
          <h3
            class="css-1vpdv9u"
          >
            岡山大学 経済学部
          </h3>
          <p
            class="css-1vf9ext"
          >
            2014/4 ~ 2018/3
          </p>
          <a
            class="chakra-link css-1lyjkv1"
            href="https://www.okayama-u.ac.jp/"
            rel="noopener noreferrer"
            target="_blank"
          >
            大学のHPを見る
            <svg
              class="chakra-icon css-soaf0h"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <g
                fill="none"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-width="2"
              >
                <path
                  d="M18 13v6a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h6"
                />
                <path
                  d="M15 3h6v6"
                />
                <path
                  d="M10 14L21 3"
                />
              </g>
            </svg>
          </a>
        </div>
      </div>
      <div
        class="chakra-stack css-cx70wh"
      >
        <div
          class="chakra-stack css-3c05cu"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 version=%271.1%27 width=%27128%27 height=%27128%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="Yutaro Adachiのプロフィール画像"
              class="circle-image"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
          <div
            class="chakra-stack css-igigla"
          >
            <h2
              class="css-553wiy"
            >
              Yutaro Adachi
            </h2>
            <p
              class="css-1vf9ext"
            >
              フロントエンドエンジニア
            </p>
          </div>
          <div
            class="chakra-stack css-a9v878"
          >
            <a
              class="chakra-link css-f4h6uy"
              href="https://github.com/yutaroadachi"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                class="chakra-icon css-1fmr1qf"
                fill="none"
                focusable="false"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 19c-5 1.5-5-2.5-7-3m14 6v-3.87a3.37 3.37 0 0 0-.94-2.61c3.14-.35 6.44-1.54 6.44-7A5.44 5.44 0 0 0 20 4.77 5.07 5.07 0 0 0 19.91 1S18.73.65 16 2.48a13.38 13.38 0 0 0-7 0C6.27.65 5.09 1 5.09 1A5.07 5.07 0 0 0 5 4.77a5.44 5.44 0 0 0-1.5 3.78c0 5.42 3.3 6.61 6.44 7A3.37 3.37 0 0 0 9 18.13V22"
                />
              </svg>
            </a>
            <a
              class="chakra-link css-f4h6uy"
              href="https://twitter.com/a_da_chi_"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                class="chakra-icon css-1fmr1qf"
                fill="none"
                focusable="false"
                height="1em"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M23 3a10.9 10.9 0 0 1-3.14 1.53 4.48 4.48 0 0 0-7.86 3v1A10.66 10.66 0 0 1 3 4s-4 9 5 13a11.64 11.64 0 0 1-7 2c9 5 20 0 20-11.5a4.5 4.5 0 0 0-.08-.83A7.72 7.72 0 0 0 23 3z"
                />
              </svg>
            </a>
          </div>
        </div>
        <p
          class="css-0"
        >
          大学で経済学を学んだのち、大手SIerでメンバーおよびチームリーダーとして客先常駐でのシステム開発を経験しました。この経験を通して、自分でもコードを書けるようになる必要があると感じ、エンジニアにキャリアチェンジすることを決意しました。
          <br />
          その後スタートアップでエンジニアとしてフロントエンド開発からバックエンド開発まで幅広く経験してきましたが、特にReactを中心としたフロントエンド開発が得意です。また要件定義や設計といった上流工程の経験もしてきたため、一人称で開発に携わることができると自負しております。
        </p>
      </div>
    </div>
  </div>
</div>
`;
